plugins/modules/vmware_guest_vnc.py

```diff
@@ -11,6 +11,11 @@
 DOCUMENTATION = r'''
 ---
 module: vmware_guest_vnc
+deprecated:
+  removed_at_date: '2022-10-15'
+  why: VNC has been removed in 7.0 U(https://docs.vmware.com/en/VMware-vSphere/7.0/rn/vsphere-esxi-vcenter-server-70-release-notes.html#productsupport).
```

Comment:
How about also adding the reason for the removing date to why? (no need?)
e.g. vCenter support will be End of General Support at 2022-10-15

Reply:
I've added this information, but also removed the link to the release notes because I don't want to make the explanation why this module is deprecated too long.